TableViewKit/TableViewKitDataSource.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+open class TableViewKitDataSource: NSObject, UITableViewDataSource {
+
+    open unowned var manager: TableViewManager
+    open var sections: ObservableArray<Section> { return manager.sections }
```

Comment:
This `var` is used in private scope. Can you use `private` instead `open`? 
The same for `manager`